test/CMakeLists.txt

```diff
@@ -24,12 +24,12 @@ if(SPIRV_TOOLS_FOUND)
 endif()
 
 if(NOT SPIRV_TOOLS_SPIRV_AS)
-  message(WARNING "spirv-as not found! SPIR-V assembly tests will not be run.")
+  message(FATAL_ERROR "spirv-as not found!")
   set(SPIRV_TOOLS_SPIRV_AS_FOUND False)
```

Comment:
Should we also remove `SPIRV_TOOLS_SPIRV_AS_FOUND`?

If so, `config.spirv_tools_have_spirv_as` will need to be removed from `lit.site.cfg.py.in`, too.